Lib9c.Models/States/StakeState.cs

```diff
@@ -15,7 +15,7 @@ namespace Lib9c.Models.States;
 public record StakeState : IBencodable
 {
     public const string StateTypeName = "stake_state";
-    public const int StateTypeVersion = 2;
+    public const int StateTypeVersion = 3;
```

Comment:
To implement `StakeState` completely, it should set `StateTypeVersion` from its state.

Reply:
Umm, could we also remove this validation?

```

        if (
            l[0] is not Text stateTypeNameValue
            || stateTypeNameValue != StateTypeName
            || l[1] is not Integer stateTypeVersionValue
            || stateTypeVersionValue.Value != StateTypeVersion
        )
        {
            throw new ArgumentException(
                nameof(bencoded),
                $"{nameof(bencoded)} doesn't have valid header."
            );
        }

```

Reply:
However, merge first